DXMainClient/Online/Channel.cs

```diff
@@ -259,13 +259,23 @@ public void SendChatMessage(string message, IRCColor color)
                 "PRIVMSG " + ChannelName + " :" + colorString + message);
         }
 
-        public void SendCTCPMessage(string message, QueuedMessageType qmType, int priority)
+        /// <summary>
+        /// 
```

Comment:
Probably forgot to write? This may be helpful: https://en.wikipedia.org/wiki/Client-to-client_protocol

Reply:
Not necessarily. There was no summary on this function to begin with. I just know how the replace functionality should work for it. So, I added the summary to explain the param. I didn't want to write a summary without a full knowledge of the underlying functionality of the messaging.